app/views/posts/index.html.erb

```diff
@@ -1 +1,3 @@
-<h1>These are the posts for the user</h1>
\ No newline at end of file
+<%= render @user%>
+
+<%= render partial: "posts/postandcomments", collection:@posts %>
```

Comment:
Kindly use the `most_recent_comments` method you have defined in the **Post** model to get the five recent comments. This is the exact purpose of this method since we have to only display 5 recent comments on the post index page. It is one of the project requirements. Here you can see in the screenshot, it is displaying more than 5 comments. 
![image](https://github.com/MosDevx/BlogApp/assets/33268927/fb3a5623-764b-4d64-8d72-8f6d3024561d)

Reply:
done